libvmaf/src/model.h

```diff
@@ -43,6 +43,11 @@ typedef struct {
     VmafDictionary *opts_dict;
 } VmafModelFeature;
 
+typedef struct {
+    double x;
+    double y;
+} Point;
```

Comment:
Might be a good idea to give this type a Vmaf... prefix. This is to avoid collisions with symbols/types in other libraries.

Reply:
Good call.